src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/PopulateSourceDbMetadataForImport.sql

```diff
@@ -240,8 +240,18 @@ VALUES
  ,('Assessor','Organisations','dbo','[Id],[CreatedAt],[DeletedAt],[EndPointAssessorOrganisationId],[Status],[UpdatedAt],[OrganisationTypeId],[ApiEnabled],[ApiUser]','','[EndPointAssessorName],[EndPointAssessorUkprn]',0,1,
     'select [Id],[CreatedAt],[DeletedAt],[EndPointAssessorName],[EndPointAssessorOrganisationId],[EndPointAssessorUkprn],[Status],[UpdatedAt],[OrganisationTypeId],[ApiEnabled],[ApiUser] From [dbo].[Organisations]',
     'Assessor_Organisations')
-
-
+ ,('Assessor','OrganisationStandardDeliveryArea','dbo','[Id],[OrganisationStandardId],[DeliveryAreaId]','','[Comments],[Status]',0,1,
+    'select [Id],[OrganisationStandardId],[DeliveryAreaId],[Comments],[Status] From [dbo].[OrganisationStandardDeliveryArea]',
```

Comment:
If comments are not a requirement then it might be better to exclude in all tables

Reply:
Fix the issue with the pipeline and now moved all comments columns to included columns list, there is no PII data in comments and better to include